tests/unit/test_dummy.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Universität Tübingen, DKFZ and EMBL
```

Comment:
It would be still great if we could run, if we could run the tests scripts in an automated fashion within the CI.

Reply:
Sure, but this is something for 1506.
Wanted to remove the randomness and have state checks in place first